ansible/roles/qemu-user-static/tasks/main.yml

```diff
@@ -23,6 +23,14 @@
     group: root
   register: qemu_user_static
 
+- name: Mask systemd-binfmt service
+  become: true
+  systemd:
+    name: systemd-binfmt
+    state: stopped
+    masked: yes
+  register: mask_systemd_binfmt
```

Comment:
I don't think this is needed if you don't use it later: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#registering-variables

Reply:
Removed.